multilayer_EC_glioma_2023/Supra_adjacencymatrix_github.m

```diff
@@ -0,0 +1,226 @@
+
+% Manuscript Multilayer EC in FPN - executive functioning in glioma 
+% Final script made in december 2021 to make adjacency matrices from MEG
+% data using the phase lag index (PLI) and Minimum Spanning Tree (MST). 
+% The output serves as input for the Python multilayer script (https://github.com/multinetlab-amsterdam/data_analysis/tree/Multilayer/Multilayer)
+
+%   Description of the goal and usage of the script
+
+%%author__ = Marike van Lingen and Lucas Breedt 
+%%contact__ = m.vanlingen@amsterdamumc.nl l.breedt@amsterdamumc.nl
+%%date__ = 2021/1/12 approximately, adjusted from earlier scripts 
+%%status__ = finished 
+
+
+%%%%%%%%%%%%%%%%%%%%
+% Review History   %
+%%%%%%%%%%%%%%%%%%%%
+
+%%% Reviewed by LC Breedt 2023/05/09 
+
+%%%%%%%%%%%%%%%%%%%%
+% Requirements     %
+%%%%%%%%%%%%%%%%%%%%
+
+%%% Other m-files required:
+% other m-files that are necessary to run the script, i.e. all functions etc.
+ 
+% /data/KNW/m.vanlingen/m2b_manuscript/fft_filt_BNA.m
```

Comment:
maybe these don't need to be explicit paths in the server?

Reply:
Totally right, didn't forgot to change those.